### PR TITLE
Fix documentation #21

### PR DESCRIPTION
**Pull request by alehodro**
## PR title

* added new audio, video, image, avatar, contact, document, media, user description pages;
* deleted unnecessary botcommand.png image;
* updated retrieveUpdates.png image;
* updated contribution, ChangeGroupTitle, ChangeGroupAvatar, ChangeGroupDescription, channeladminadded, channeladmindeleted, channelpermissiongranted, commandrequest, DeleteMessage, EditContainerMessage, EditMessage, endpoints, foreclosed, formmessagesent, form submitted, ForwardMessage, getting update, getWebhookInfo, inlinecommandselected, input contact, InputMedia, KickFromGroup, make requests, message, message edited, messageforwardmetadata,messageidassigned, newchannelsubscriber, quickbuttoncommand, quickbuttonselected, replycommand, retrieveUpdate, SendContactMessage, SendContainerMessage, SendMessage, SendUiState, SetWebhook, uistate, update response ( fixed grammar, enhanced content, added web-links);
* replaced all exact uuids in samples with "Uuid_value"
* replaced all exact phone numbers in samples with "+7**********"

Reviewers:
* @navi89mai
* @mustafin
* @vseko

### CODE REVIEW

File: docs/pages/model/SendUiState.md [hunk 1]
```diff
@@ -29,38 +28,102 @@ https://messapi.btsdapps.net/bot/v1/updates/
...
-         }
-      }
-   ]
-}
+{
+ 	 "commands": [{
+ 	 	"type": "SendUiState",
+ 	 	"recipient": {
+ 	 		"type": "USER",
+ 	 		"id": "ga018bae-e3e7-11e8-a458-7a609eaf8200"
```
Comment (yerassyl94) [inline docs/pages/model/SendUiState.md]: please remove personal token

File: docs/pages/model/SendUiState.md [hunk 2]
```diff
@@ -29,38 +28,102 @@ https://messapi.btsdapps.net/bot/v1/updates/
...
+ 	 	},
+ 	 	"dialog": {
+ 	 		"type": "USER",
+ 	 		"id": "ga018bae-e3e7-11e8-a458-7a609eaf8200"
```
Comment (yerassyl94) [inline docs/pages/model/SendUiState.md]: please remove personal token
```suggestion
 	 		"id": "dialog_id"
```

File: docs/pages/model/audio.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+| name  | true |String |  Backend name of audio file|
+| mimeType  | true |String| [mimeType](https://www.freeformatter.com/mime-types-list.html) of video file |
+| size  | true |Int| File size, bytes |
+
+## Sample object
+
+```
+{  
+      "type":"Audio",
+      "fileId":"361a63f1-abbd-4026-b4fb-96e1d70c5bd1",
```
Comment (yerassyl94) [inline docs/pages/model/audio.md]: please remove token
```suggestion
      "fileId": <your_file_id>,
```

File: docs/pages/model/avatar.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+
+
+## Sample object
+
+```
+{  
+   "full":{  
+      "type":"Image",
+      "fileId":"361a63f1-abbd-4026-b4fb-96e1d70c5bd1",
```
Comment (yerassyl94) [inline docs/pages/model/avatar.md]: please remove id

File: docs/pages/model/avatar.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+      "mimeType":"JPEG",
+      "name":"test_avatar.jpeg",
+      "size":1024,
+      "width":800,
+      "height":600
+   },
+   "small":{  
+      "type":"Image",
+      "fileId":"382f03e6-12cd-4ef4-a88e-95a4ad05d350",
```
Comment (yerassyl94) [inline docs/pages/model/avatar.md]: please remove id

File: docs/pages/model/contact.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+
+
+## Sample object for RegisteredContact
+
+```
+{  
+      "type":"RegisteredContact",
+      "user": {
+                        "id" : "f1a6b389-40e2-4318-adbc-3b38c57bf1f4",
```
Comment (yerassyl94) [inline docs/pages/model/contact.md]: please remove id

File: docs/pages/model/contact.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+                        "firstName" : "Alex",
+                        "lastName" : "Key",
+                        "phoneNumber" : "+77077661234",
```
Comment (yerassyl94) [inline docs/pages/model/contact.md]: please remove personal info

File: docs/pages/model/contact.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+                        "userName" : "@test",
+                        "avatar" : {
+                          "full" : {
+                            "type" : "Image",
+                            "fileId" : "361a63f1-abbd-4026-b4fb-96e1d70c5bd1",
```
Comment (yerassyl94) [inline docs/pages/model/contact.md]: please remove id

File: docs/pages/model/contact.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+                            "mimeType" : "JPEG",
+                            "name" : "test_avatar.jpeg",
+                            "size" : 1024,
+                            "width" : 800,
+                            "height" : 600
+                          },
+                          "small" : {
+                            "type" : "Image",
+                            "fileId" : "382f03e6-12cd-4ef4-a88e-95a4ad05d350",
```
Comment (yerassyl94) [inline docs/pages/model/contact.md]: please remove id

File: docs/pages/model/contact.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+   }
+```
+
+## Sample object for UnregisteredContact
+
+```
+{  
+      "type":"UnregisteredContact",
+      "firstName": "Tom",
+      "phoneNumber":"+77018981221"
```
Comment (yerassyl94) [inline docs/pages/model/contact.md]: please remove personal info

File: docs/pages/model/document.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
+
+## Sample object
+
+```
+{  
+      "type":"Document",
+      "fileId":"361a63f1-abbd-4026-b4fb-96e1d70c5bd1",
```
Comment (yerassyl94) [inline docs/pages/model/document.md]: please remove id

File: docs/pages/model/formmessagesent.md [hunk 12]
```diff
@@ -18,37 +18,30 @@ folder: Types
...
-                   "firstName" : "TestfirstName"
-                 },
-                 "message" : "test_message",
-                 "messageId" : "c55330fe-9d43-4f4f-b068-f6d50c0750c8",
-                 "additionalMetadata" : "some_metadata"
-               }
+                           "updateId": "f8095479-d7a6-4122-b4e1-f670b058069f",
+                           "formId": "",
+                           "dialog": {
+                               "id": "3a17518d-63d4-4d63-9c46-9c312588779a",
```
Comment (yerassyl94) [inline docs/pages/model/formmessagesent.md]: please remove id

File: docs/pages/model/formmessagesent.md [hunk 13]
```diff
@@ -18,37 +18,30 @@ folder: Types
...
+                               "type": "USER"
+                           },
+                           "sender": {
+                               "id": "3a17518d-63d4-4d63-9c46-9c312588779a",
+                               "type": "USER"
+                           },
+                           "message": "+77057252478",
+                           "messageId": "556c0421-1299-11ea-9b4c-02af8e11935e",
+                           "additionalMetadata": "{\"data_template_result\":\"77057252478\",\"private_data\":{\"value\":{\"phone_number\":\"+77077221478\",\"metadata\":\"XXX\"}},\"hidden_metadata\":\"\"}",
```
Comment (yerassyl94) [inline docs/pages/model/formmessagesent.md]: please remove personal info

File: docs/pages/model/image.md [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+
+
+
+
+## Sample object
+
+```
+{  
+      "type":"Image",
+      "fileId":"361a63f1-abbd-4026-b4fb-96e1d70c5bd1",
```
Comment (yerassyl94) [inline docs/pages/model/image.md]: please remove id

File: docs/pages/model/inputmedia.md [hunk 15]
```diff
@@ -18,12 +18,12 @@ folder: Types
...
 | name | true |  String |LocalID length should be <= 255  | Backend name of a file |
-| mediaType  | true |  InputMediaType | -| Type of media file (DOCUMENT, IMAGE, VIDEO, AUDIO) |
+| mediaType  | true |  string | -| Type of media file must be set: "DOCUMENT","IMAGE","VIDEO","AUDIO".|
 
 ## Sample object
 
 ```
-{
+    {
  		"fileId":"08d5251b-0b86-11ea-9162-ee73f85aba69",
```
Comment (yerassyl94) [inline docs/pages/model/inputmedia.md]: please remove id

File: docs/pages/model/quickbuttoncommand.md [hunk 16]
```diff
@@ -53,82 +64,120 @@ This type of action  may invoke native mobile device functions as well.</p>
...
+ 	 			},
+ 	 			{
+ 		 			"caption":"Open peer",
+ 		 			"action":"QUICK_FORM_ACTION",
+ 		 			"metadata":"{\r\n   \"action\" : \"open_peer\",\r\n   \"data_template\" : \"@MasterService\"\r\n}"
+ 	 			},
+ 	 			{
+ 		 			"caption":"Call me maybe",
+ 		 			"action":"QUICK_FORM_ACTION",
+ 		 			"metadata":"{\r\n   \"action\" : \"redirect_call\",\r\n   \"data_template\" : \"+77072771478\"\r\n}"
```
Comment (yerassyl94) [inline docs/pages/model/quickbuttoncommand.md]: please remove personal info

File: docs/pages/model/user.md [hunk 17]
```diff
@@ -0,0 +1,58 @@
...
+
+
+## Sample object
+
+```
+        {
+          "id" : "f1a6b389-40e2-4318-adbc-3b38c57bf1f4",
+          "firstName" : "Alex",
+          "lastName" : "Key",
+          "phoneNumber" : "+77077661234",
```
Comment (yerassyl94) [inline docs/pages/model/user.md]: please remove personal info

File: docs/pages/model/user.md [hunk 18]
```diff
@@ -0,0 +1,58 @@
...
+          "userName" : "@test",
+          "avatar" : {
+            "full" : {
+              "type" : "Image",
+              "fileId" : "361a63f1-abbd-4026-b4fb-96e1d70c5bd1",
```
Comment (yerassyl94) [inline docs/pages/model/user.md]: please remove id

File: docs/pages/model/user.md [hunk 19]
```diff
@@ -0,0 +1,58 @@
...
+              "mimeType" : "JPEG",
+              "name" : "test_avatar.jpeg",
+              "size" : 1024,
+              "width" : 800,
+              "height" : 600
+            },
+            "small" : {
+              "type" : "Image",
+              "fileId" : "382f03e6-12cd-4ef4-a88e-95a4ad05d350",
```
Comment (yerassyl94) [inline docs/pages/model/user.md]: please remove id

File: docs/pages/model/video.md [hunk 20]
```diff
@@ -0,0 +1,45 @@
...
+
+
+
+
+## Sample object
+
+```
+{  
+      "type":"Video",
+      "fileId":"361a63f1-abbd-4026-b4fb-96e1d70c5bd1",
```
Comment (yerassyl94) [inline docs/pages/model/video.md]: please remove id

**Review by yerassyl94 — CHANGES_REQUESTED** [review]
please resolve comments

File: docs/pages/model/avatar.md (hunk 4 above)
Comment (alehodro) [reply docs/pages/model/avatar.md]: This is randomly generated uuid

File: docs/pages/model/avatar.md (hunk 5 above)
Comment (alehodro) [reply docs/pages/model/avatar.md]: This is randomly generated uuid

File: docs/pages/model/contact.md (hunk 6 above)
Comment (yerassyl94) [reply docs/pages/model/contact.md]: it is a randomly generated uuid

File: docs/pages/model/contact.md (hunk 7 above)
Comment (alehodro) [reply docs/pages/model/contact.md]: Fixed

File: docs/pages/model/contact.md (hunk 10 above)
Comment (alehodro) [reply docs/pages/model/contact.md]: Fixed

File: docs/pages/model/formmessagesent.md (hunk 13 above)
Comment (alehodro) [reply docs/pages/model/formmessagesent.md]: Fixed

File: docs/pages/model/quickbuttoncommand.md (hunk 16 above)
Comment (alehodro) [reply docs/pages/model/quickbuttoncommand.md]: Fixed

File: docs/pages/model/user.md (hunk 17 above)
Comment (alehodro) [reply docs/pages/model/user.md]: Fixed

File: docs/pages/model/formmessagesent.md [hunk 21]
```diff
@@ -9,46 +9,39 @@ folder: Types
...
+                               "id": "Uuid_value",
+                               "type": "USER"
+                           },
+                           "sender": {
+                               "id": "Uuid_value",
+                               "type": "USER"
+                           },
+                           "message": "+7**********",
+                           "messageId": "Uuid_value",
+                           "additionalMetadata": "{\"data_template_result\":\"7**********\",\"private_data\":{\"value\":{\"phone_number\":\"+77077221478\",\"metadata\":\"XXX\"}},\"hidden_metadata\":\"\"}",
```
Comment (yerassyl94) [inline docs/pages/model/formmessagesent.md]: phone number)

**Review by yerassyl94 — CHANGES_REQUESTED** [review]

File: docs/pages/model/formmessagesent.md (hunk 21 above)
Comment (alehodro) [reply docs/pages/model/formmessagesent.md]: fixed

**Review by yerassyl94 — APPROVED** [review]

**Pull request merged**